components/general/LatestNews.tsx

```diff
@@ -13,7 +13,9 @@ const LatestNews = () => {
         >
           LATEST NEWS
         </h1>
-        <Button variant="secondary">SEE ALL</Button>
+        <Button variant="secondary" isBlock={true}>
+          SEE ALL
+        </Button>
```

Comment:
Do we need a block style on this "see all" button?

Reply:
Its parent is a row direction flex style. Cause we just find that adding align-self:flex-start to Button is a solution. The following result is without block(align-self: auto):
<img width="1604" alt="image" src="https://user-images.githubusercontent.com/110534329/220079038-263c4d9c-3fcf-4444-9e82-49d5021f6297.png">